src/logging.service.ts

```diff
@@ -21,12 +21,20 @@ export class LoggingService extends ConsoleLogger implements LoggerService {
     super();
   }
 
+  protected composeEntry(severity: Severity, message: any): object {
```

Comment:
I would suggest adding the type for the log entry with the minimum set of required fields:

```
interface LogEntry {
  message: unknown;
  severity: Severity;
}
```

Having `object` allows specifying any fields which may introduce bugs in the future. With strict type, the consumer apps that decide to extend this `LoggingService` will be forced to specify the required fields like `message` and `severity`:

```
class MyCustomerLogger extends LoggingService {
   ...
   protected composeEntry(severity: Serverity, message: unknown): LogEntry {
      return { message, severity }
```

Reply:
Also, would be nice to export both `Severity` and `LogEntry` to use outside the outside of the package:

```
import { LoggingService, Severity, LogEntry } from '@pzwik/nestjs-gcp-logger';
```